### DISCUSSION TITLE
Edit Schedule of Cleaning

### DISCUSSION THREAD

**Discussion by Vippis2000** [Tasshack/dreame-vacuum#594] [General]
Hi,
how do I edit the schedule of the cleaning tasks like in Dreame App?

**Comment by Tasshack** [comment]
Currently it is not possible via the integration since Home Assistant can already do that via automations.

**Reply by mvereecken** [reply]
Is it possible you use the schedules I defined in the Dreame app that way? Because a schedule is also a bunch of settings (clean/mop/what rooms/...). Or does this mean you have to create your own settings away from the app and do it that way?

**Reply by Tasshack** [reply]
Not possible yet.

**Reply by mvereecken** [reply]
Thanks for your response, and for all the work you put into this. Schedule editing might be a lot of work, but maybe it would be nice to start with the possibility to call the schedules as a service, t.i. enable/disable a schedule, or send a new start time. There are cards like the scheduler card that we then can use to change the schedule from within HA. Just an idea.
